app/utils/forking-store-helper.js

```diff
@@ -0,0 +1,15 @@
+import { ForkingStore } from '@lblod/ember-submission-form-fields';
+
+export default class ForkingStoreHelper {
```

Comment:
Nitpick: Is there a benefit of using a class + static methods over exported util functions?

AFAIK, this setup has the downside that it can't be properly tree-shaken if we would switch to the new Embroider build system in the future.

Reply:
Defining a class for it is Like the context for the method that will be called. For example you cannot create a method `createFromJson(json)` because you don't know what this will do. If you make a class in context of an Employee Controller for example you can call `Employee.createFromJson()` this example is actuallly not so great because you will not do this but I think this makes the reason behind it more clear?

Also I understand why we want to export these as functions and not as classes (after reading the `tree-shaken`)

Before I was writing symphony backend code so maybe it comes from there :)

=> will change it to functions in the future when I want to make a util